src/components/Books/Author.vue

```diff
@@ -0,0 +1,106 @@
+            <img src="https://randomuser.me/api/portraits/women/44.jpg" alt="User Avatar" class="w-24 h-24 rounded-full" />
+            <h1 class="text-2xl">What Do You Think?</h1>
+            <div class="flex justify-between">
+                <div class="text-center">
+                    <div class="flex items-center text-xl space-x-2">
+                        <i class="fas fa-star text-yellow-400"></i>
+                        <i class="fas fa-star text-yellow-400"></i>
+                        <i class="fas fa-star text-yellow-400"></i>
+                        <i class="fas fa-star text-gray-300"></i>
+                        <i class="fas fa-star text-gray-300"></i>
+                    </div>
+                    <h1 class="pt-4 text-lg">Rate this Book</h1>
+                </div>
+
+                <button class="ml-10 mt-5 bg-dark_blue text-white h-fit p-2 px-4 rounded-2xl hover:bg-light_blue">Write a Review</button>
+            </div>
+        </div>
+        <div class="absolute -bottom-2 right-64 b-0 w-3/5 h-0.5 bg-dark_blue"></div>
+
+        <div class="mt-12 p-6">
+            <h2 class="text-6xl font-tangerine mb-4 text-center">Community Reviews</h2>
+            <div class="flex items-center mb-4">
+                <div class="text-lg text-center">
+                    <span v-for="n in 5" :key="n">
+                        <i :class="['text-yellow m-2', n <= 4 ? 'fa-solid fa-star' : 'fa-regular fa-star']"></i>
+                    </span>
+                </div>
+                <span class="ml-2 text-xl">4.5</span>
+                <span class="ml-4 text-gray-500">(654 Ratings • 756 Reviews)</span>
+            </div>
+            <div class="w-full space-y-5">
+                <div class="flex items-center" v-for="(rate, index) in rates" :key="index">
+                    <div class="text-sm">
+                        <span class="mr-1">{{ rate.star }}</span
+                        >stars
+                    </div>
+                    <div class="w-full bg-gray h-2 rounded-full mx-2">
+                        <div
+                            class="bg-green h-2 rounded-full"
+                            :style="{ width: rate.width }"
+                            :class="[rate.star === 1 ? 'bg-red' : 'bg-green', rate.star === 2 ? 'bg-orange_dark' : 'bg-green']"
+                        ></div>
+                    </div>
+                    <span class="text-sm">{{ rate.percent }}</span>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script setup>
+import { ref } from 'vue'
+import 'animate.css'
+import FollowAuthorButton from '@/components/Books/FollowAuthorButton.vue'
+// Reactive properties
+
+const rates = ref([
+    { star: 5, width: '80%', percent: '80%' },
+    { star: 4, width: '60%', percent: '60%' },
+    { star: 3, width: '40%', percent: '40%' },
+    { star: 2, width: '20%', percent: '20%' },
+    { star: 1, width: '10%', percent: '10%' },
+])
```

Comment:
You are not changing this rates variable in your code, so, you don't really need a reactive variable.
```suggestion
const rates = [
    { star: 5, width: '80%', percent: '80%' },
    { star: 4, width: '60%', percent: '60%' },
    { star: 3, width: '40%', percent: '40%' },
    { star: 2, width: '20%', percent: '20%' },
    { star: 1, width: '10%', percent: '10%' },
]
```